core/unit_test/TestGraph.hpp

```diff
@@ -174,6 +174,40 @@ TEST_F(TEST_CATEGORY_FIXTURE(graph), can_instantiate_only_once) {
   }
 }
 
+// This test submits on an execution space instance different from the
+// one passed to the Kokkos::Graph constructor.
+TEST_F(TEST_CATEGORY_FIXTURE(graph),
+       submit_onto_another_execution_space_instance) {
+// FIXME_HIP For ROCm 5.2, the graph implementation is bugged, and Kokkos
```

Comment:
Fine but noting that the FIXME is not quite a HIP thing but seems to denote a bug in the fallback implementation